Dockerfile

```diff
@@ -1,13 +1,18 @@
 FROM node:8-alpine
+
+COPY . /app
```

Comment:
Why move this copy to the top? it will make layer re-use really bad.

Reply:
Hmm, good point. I didn't consider that.

Reply:
Yeah I thought about doing that that but inlining isn't so bad. At least it's all in one place. It's not like we're going to re-use the script anywhere else.